content/chainguard/chainguard-images/getting-started/wordpress.md

```diff
@@ -0,0 +1,305 @@
+  document-root:
+```
+
+Only the `app` service has changed in this example. We've added a `build` section that references the custom Dockerfile, and we've set the `UID` build argument to `1000` by default. This can be overwritten at runtime when you call `docker compose up`. We've also added a volume share to persist the contents of the `wp-content` folder to the host machine.
+
+To build your custom image and pass along your own UID as build argument, run:
+
+```shell
+docker-compose build --build-arg UID=$(id -u) app
+```
+
+You should get output indicating that the image was successfully built. Now you can get your environment up with:
+
+```shell
+docker-compose up
+```
+
+Once the environment is up and running, you can access your WordPress installation from your browser at `localhost:8000`.
+
+If you go to another terminal window and check the contents of the `02-customizing/wp-content` folder, you'll notice that it was populated with the default WordPress themes and plugins:
+
+```shell
+❯ ls -la wp-content
+total 24
+drwxrwxr-x  4 erika erika 4096 Jul 18 21:16 .
+drwxrwxr-x  3 erika erika 4096 Jul 18 21:15 ..
+-rw-rw-r--  1 erika erika   14 Jul 18 21:05 .gitignore
+-rw-r--r--  1 erika 65533   28 Jan  1  1970 index.php
+drwxr-xr-x  2 erika 65533 4096 Jul 18 21:16 plugins
+drwxr-xr-x 16 erika 65533 4096 Jul 18 21:16 themes
+```
+
+This is only possible because of the custom Dockerfile we created, which added a `wordpress` user with the same UID as the local user on the host machine.
+
+You can now install new themes and plugins, and they will persist between container rebuilds.
+
+To stop the services, type `CTRL+C` in the terminal where the services are running, and then run:
+
+```bash
+docker-compose down
+```
+
+In the next example, we'll see how you can create a distroless WordPress runtime for your production environment.
+
+## Example 3: Using the Distroless Variant of the WordPress Image
+This demo uses a multi-stage Docker build to create a final Distroless image to improve overall security. The Distroless image contains the necessary dependencies to run WordPress and won't allow for new package installations or shell access, reducing the image attack surface.
+
+The main difference here is that we're calling the entrypoint script at **build time** instead of run time. This is done to ensure the image is self-contained and doesn't rely on volumes set up within the host machine in order to work. Any customizations should be included in the `wp-content` folder that will be copied to the image at build time. Although this increases final image size due to the inclusion of custom content at build time, it limits what can be changed or added to the image once it's built.
+
+This demo includes a theme ([cue](https://wordpress.org/themes/cue/), a simple blogging theme) and a plugin ([imsanity](https://wordpress.org/plugins/imsanity/), a popular plugin used to resize images) to demonstrate how to include custom content in the image.
```

Comment:
```suggestion
This demo includes a theme ([Cue](https://wordpress.org/themes/cue/), a simple blogging theme) and a plugin ([Imsanity](https://wordpress.org/plugins/imsanity/), a popular plugin used to resize images) to demonstrate how to include custom content in the image.
```
It looks like these two plugins are stylized with capitals at the links you share. I would leave it uncapitalized if that's how it's usually written elsewhere, but I am not familiar with these projects, so you'd have better judgement on that